MessageConstructors.yml

```diff
@@ -298,19 +298,14 @@ msgBoxed:
       - define buttonThis '"text":"1"'
       }
       else {
-      - define iconThis 'i@human_skull[display_name=<proc[msgCentered].context[19|Previous Page]>]'
-      - define hoverThis '{<def[iconThis].json>}'
-      - define clickThis '/%command% <def[page].sub[1]>'
-      - define buttonThis '"text":"%page%","clickEvent":{"action":"run_command","value":"%clickThis%"},"hoverEvent":{"action":"show_item","value":"%hoverThis%"}'
-      }
+     - define clickThis '/%command% <def[page].sub[1]>'
+     - define buttonThis '"text":"%page%","clickEvent":{"action":"run_command","value":"%clickThis%"},"hoverEvent":{"action":"show_text","value":"<gold>Previous Page"}'      }
     - ^if %page% == %pageTotal% {
       - define buttonNext '"text":"%pageTotal%"'
       }
       else {
-      - define iconNext 'i@human_skull[display_name=<proc[msgCentered].context[19|Next Page]>]'
-      - define hoverNext '{<def[iconNext].json>}'
       - define clickNext '/%command% <def[page].add[1]>'
-      - define buttonNext '"text":"%pageTotal%","clickEvent":{"action":"run_command","value":"%clickNext%"},"hoverEvent":{"action":"show_item","value":"%hoverNext%"}'
+      - define buttonNext '"text":"%pageTotal%","clickEvent":{"action":"run_command","value":"%clickNext%"},"hoverEvent":{"action":"show_text","value":"<gold>Next Page"}'      
```

Comment:
Format is broken on this line and 302, also item functionality was deleted rather than patched :(